CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/fedora-modularity/meta-test-family/issues/new). Be sure to include a **descriptive title and clear description** and please include **as many details as possible** to help maintainers reproduce the issue and resolve it faster. If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). When you are creating an enhancement issue, **use a clear and descriptive title** and **provide a clear description of the suggested enhancement** in as many details as possible.
+
+### Submitting changes
+
+To submitt changes, please send a [GitHub Pull Request](https://github.com/fedora-modularity/meta-test-family/pull/new/devel). Before submitting a PR, please read the [Styleguides](#styleguides) to know more about coding conventions used in this project. Also we will appreciate if you check your code with pylint and pyflakes. Always create a new branch for each pull request to avoid intertwingling different features or fixes on the same branch. Always do "git pull --rebase" and "git rebase" vs "git pull" or "git merge".
+
+> **Note:** We are aware of [current syntax and style code errors](https://github.com/fedora-modularity/meta-test-family/issues/21) and working on fixing them.
+
+## Styleguides
+
+### Git Commit Messages
+
+  * Use the present tense ("Add feature" not "Added feature")
+  * Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+  * Limit the first line to 72 characters or less
+
+      ```
+      $ git commit -m "A brief summary of the commit
+      >
+      > A paragraph describing what changed and its impact."
+      ```
+
+  * Reference issues and pull requests liberally after the first line
+  * When only changing documentation, include `[ci skip]` in the commit description
+  * Consider starting the commit message with an applicable emoji:
+      * :art: `:art:` when improving the format/structure of the code
+      * :racehorse: `:racehorse:` when improving performance
+      * :memo: `:memo:` when writing docs
+      * :bug: `:bug:` when fixing a bug
+      * :fire: `:fire:` when removing code or files
+      * :green_heart: `:green_heart:` when fixing the CI build
+      * :white_check_mark: `:white_check_mark:` when adding tests
+      * :arrow_up: `:arrow_up:` when upgrading dependencies
+      * :arrow_down: `:arrow_down:` when downgrading dependencies
+      * :shirt: `:shirt:` when removing linter warnings
+
```

Comment:
I've never used these emoji, just RFE: as mark, that it is not bug fix. or WIP:, as this is not ready to merge. it is work in progress (CI is started based on PR, so that it is important)
But I'm not against. If you find this usefull. And I would like to add also these Two there,